### PR TITLE
Update QA section

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
cc @nuritzi

**Comment by nuritzi** [comment]
@Hixie added some comments -- back to you!

**Review by nuritzi — COMMENTED** [review]
Added some comments!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -75,9 +75,13 @@ that increase our test coverage, writing tests for issues others have filed, all
 are really valuable contributions to open source projects.
 
 If this interests you, you can jump in and submit bugs without needing anyone's permission!
```
[the remark this answers is not in the document]
Comment (Hixie) [reply CONTRIBUTING.md:77]: I'll go through and change "bugs" to "bug reports" where appropriate.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -75,9 +75,13 @@ that increase our test coverage, writing tests for issues others have filed, all
...
-The #hackers-tests channel on our [Discord server](https://github.com/flutter/flutter/wiki/Chat)
-is a good place to talk about what you're doing. If you want to contribute test cases, you
-can also submit PRs, see the next section for how to set up your development environment.
+The #qa channel on our [Discord server](https://github.com/flutter/flutter/wiki/Chat)
+is a good place to talk about what you're doing. We're especially eager for QA testing when
+we announce a beta release, see https://github.com/flutter/flutter/wiki/Quality-Assurance for
```
[the remark this answers is not in the document]
Comment (Hixie) [reply CONTRIBUTING.md:80]: done

**Comment by Hixie** [comment]
updated, PTAL

**Review by nuritzi — APPROVED** [review]
LGTM!

**Comment by nuritzi** [comment]
@Hixie Not sure if the LGTM automatically appears in your inbox, so tagging you here!

**Comment by Hixie** [comment]
FWIW, I basically don't get GitHub notifications at all (or rather, I get _all_ the notifications, so the odds of me noticing any particular one are very low). Best way to get my attention is Discord. But for my own PRs you can assume I'll check on them regularly. :-)

Thanks for the review! I'll post the copy to flutter/flutter shortly.

**Pull request merged**